circuits/cpp/barretenberg/cpp/src/barretenberg/honk/pcs/shplonk/shplonk_single.hpp

```diff
@@ -2,25 +2,49 @@
 #include "barretenberg/honk/pcs/claim.hpp"
 #include "barretenberg/honk/pcs/commitment_key.hpp"
```

Comment:
glad we got rid of `shplonk.hpp` but could we not rename this file to `shplonk.hpp` instead (i don't see why it's named `shplonk_single.hpp`)

Reply:
done